spec/system/task_spec.rb

```diff
@@ -1,36 +1,36 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let!(:project) { create(:project) }
+  let(:task) { create(:task, project_id: project.id) }
```

Comment:
Factory側でassociationを定義することで、ここの記載も減らすことができます！
```suggestion
  let(:task) { create(:task) }
```